From mark.adamson@ftdichip.com  Tue Dec 22 11:26:30 2009
From: Mark Adamson <mark.adamson@ftdichip.com>
Date: Tue, 24 Nov 2009 09:39:10 +0000
Subject: USB: ftdi_sio: remove support for 5 and 6 data bits
To: "gregkh@suse.de" <gregkh@suse.de>
Message-ID: <8D2B66FD8A5723429FEFCD40CC13CDD1F9776EFEAA@GLAEXSV2.ftdi.local>


From: Mark Adamson <mark.adamson@ftdichip.com>

Removed CS5 and CS6 from data bits since these are not supported
in FTDI hardware.


Signed-off-by: Mark J. Adamson <mark.adamson@ftdichip.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/ftdi_sio.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

--- a/drivers/usb/serial/ftdi_sio.c
+++ b/drivers/usb/serial/ftdi_sio.c
@@ -2246,12 +2246,10 @@ static void ftdi_set_termios(struct tty_
 	}
 	if (cflag & CSIZE) {
 		switch (cflag & CSIZE) {
-		case CS5: urb_value |= 5; dbg("Setting CS5"); break;
-		case CS6: urb_value |= 6; dbg("Setting CS6"); break;
 		case CS7: urb_value |= 7; dbg("Setting CS7"); break;
 		case CS8: urb_value |= 8; dbg("Setting CS8"); break;
 		default:
-			dev_err(&port->dev, "CSIZE was set but not CS5-CS8\n");
+			dev_err(&port->dev, "CSIZE was set but not CS7-CS8\n");
 		}
 	}
 
